src/en/community/team/index.html

```diff
@@ -58,7 +58,7 @@ <h3 class="h3">Component Leads</h3>
               <td>RADOS</td>
             </tr>
             <tr>
-              <td><a class="a" href="#">Nizamudeen A</a></td>
+              <td><a class="a" href="#">Afreen Misbah</a></td>
```

Comment:
Hey Nizam, just talked about this with @afreen23. I strongly suggest that both of you add a profile picture and small bio to the bottom section :D 

BTW, could you plz fix my bio section below?:
```diff
- Ernesto is the Ceph Dashboard component lead.
+ Ernesto was the Ceph Dashboard component lead from 2020 to 2022.
```

Reply:
sorry @epuertat i forgot to come and check this PR. Yup, I'll fix the bio there and update this PR.

btw, the bottom section was reserved for maintainers isn't it?

Reply:
@afreen23 let's add a bio section at the bottom. I'll add one for myself too. The photo might be optional there but its good to leave something there so people can easily contact us.

For now we can merge this PR and take it in separately.